app/jobs/send_message.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+class SendMessage
+  def perform
+    if Meeting.tomorrow_meeting.present?
+      Meeting.tomorrow_meeting.each do |meeting|
+        message = "Hey, #{meeting.user.email}, we’re eagerly waiting to see to at the meeting #{meeting.title}, tomorrow at #{meeting.start_time.strftime("%I:%M%p")}"
```

Comment:
we need to send name of the user and not email, meeting.user.email

Reply:
I am fetching only mail from user so I wrote meeting.user.email

Reply:
okay